add EKS installation document

yuki-mt opened the pull request:
## What is this PR for?

write instructions to build rekcurd on EKS

## This PR includes

- Setup document for EKS
- Add config file to build worker nodes for EKS
   - the config file is based on [official YAML file](https://amazon-eks.s3-us-west-2.amazonaws.com/cloudformation/2018-12-10/amazon-eks-nodegroup.yaml).
    (add some rekcurd-specific settings)

## What type of PR is it?

Feature

## What is the issue?

#4 

## How should this be tested?

1. Follow instructions of the document.
2. Add the EKS cluster as Kubernetes Host in dashboard. ref: https://github.com/rekcurd/drucker-dashboard/pull/51
3. Try some operations in dashboard to handle the cluster.

keigohtr commented on docs/Installation-eks.md:
```diff
@@ -0,0 +1,64 @@
+    1. For **Choose a template**, select **Upload a template to Amazon S3.**
+    2. Select the YAML file you edited and choose **Next.**
+
+# Install Metrics Server
```
It is a trivial comment.
"#" (h1 tag) must be used only once in the document. You must not use it in several times.

keigohtr commented:
Almost LGTM.
Necessity components are
- [ ] How to setup EKS
- [ ] How to setup EKS LB/ingress
- [ ] How to setup AWS storage and mount it to the node (e.g. Using goofys in our case)
- [ ] How to label (e.g. development/staging/production) node servers
- [ ] How to setup SSH keys

yuki-mt commented:
@keigohtr 

> How to label (e.g. development/staging/production) node servers

I added the labeling setup to CloudFormation YAML file in https://github.com/rekcurd/drucker-parent/pull/16/commits/b5005769fa171696a7043e3155add782fc74eba0

> "#" (h1 tag) must be used only once in the document. You must not use it in several times.

I followed the format of kubeadm install document, so I also fixed the document in https://github.com/rekcurd/drucker-parent/pull/16/commits/56005a35cc75317d5558b7c03003a91711695c65 .

> How to setup EKS

It is in [here](https://github.com/rekcurd/drucker-parent/pull/16/files#diff-c0f9073ad7c70ed450011795f18bd944R8) (and I don't think we need to mention the same thing as the official document)

> How to setup EKS LB/ingress

It is in [here](https://github.com/rekcurd/drucker-parent/pull/16/files#diff-c0f9073ad7c70ed450011795f18bd944R40)

> How to setup AWS storage and mount it to the node (e.g. Using goofys in our case)

It is in [here](https://github.com/rekcurd/drucker-parent/pull/16/files#diff-f3943ce13c126be4e0186e2ed9eb288dR286) and [here](https://github.com/rekcurd/drucker-parent/pull/16/files#diff-a98c9db69add2229e2922a5d15419c40R63), so users do not need anything more.

> How to setup SSH keys

It is in [here](https://github.com/rekcurd/drucker-parent/pull/16/files#diff-c0f9073ad7c70ed450011795f18bd944R16)

keigohtr commented:
@yuki-mt Sorry for confusing you.
I just want to add the checkbox to clarify the items we need to cover. So please check it you already done.

yuki-mt commented:
Oh, OK. I just checked it! @keigohtr

keigohtr commented:
Thank you!
Could you create `aws` or `eks` or `amazon` directory and move `config` and `script` directories under that directory.

yuki-mt commented:
I added specific versions.
I will merge this PR after https://github.com/rekcurd/drucker-dashboard/pull/51 is released in the new version.

yuki-mt commented:
@keigohtr 
Since many documents were added in the last PR, I updated some parts such as adding EKS information to `assign-dns.md` `mount-online-storage.md` and `ssh-key.md`

Please check them. I will merge the PR if it is also approved.

keigohtr commented:
LGTM
